var/spack/repos/builtin/packages/occa/package.py

```diff
@@ -50,13 +51,16 @@ class Occa(Package):
             description='Activates support for OpenCL')
     variant('coi',   default=False,
             description='Activates support for COI')
-    depends_on('cuda', when='+cuda')
-    
+    # depends_on('cuda', when='+cuda')
```

Comment:
Try:
```
depends_on('cuda', when=(platform.system() != 'Darwin'))
```
This will fix the `spec['cuda']` issue below.  We should do a better job of fetching arch-specific binaries for things like CUDA -- then we could have a decent CUDA OSX installer.  `brew` does this pretty well with Casks, though I do not know whether they're really supposed to auto-install CUDA.